fetch.bs

```diff
@@ -5157,8 +5279,6 @@ these steps:
       <p>If one of the following is true
 
       <ul class=brief>
-       <li><p><var>response</var>'s <a for=response>type</a> is "<code>error</code>"
```

Comment:
You're still making changes to HTTP fetch rather than HTTP-network fetch or HTTP-network-or-cache fetch. This ends up impacting service workers as I pointed out before and seems unlikely to be intended. Could you please clarify this? (If you have explained this somewhere and I overlooked it, apologies. The PR is pretty long and I tried to go through the comments, but there's a lot of them.)

Reply:
As we discussed in chat, I'll move this to HTTP network fetch.

There are several `return a network error` steps in HTTP Network fetch. I think HTTPS upgrading fallback algorithm should run for all of those network errors.

So I'm planning to replace `return a network error` steps with `return <a>HTTPS upgrade fallback</a> given <var>request</var> and <var>network error</var>` in HTTP Network Fetch (including one step where an `appropriate network error` is returned instead of simply `network error`). Does that sound about right?

Reply:
No it does not. That doesn't make sense to me for the cases that fail due to streaming as those are not navigation-related.

It makes sense for _connection_ being failure.

It might make sense for the TLS client certificate dialog? Although maybe that in itself should be a fallback sign?

I also don't see how it makes sense in the canceled case.

Reply:
Moved to network fetch. Now, steps 7.1, 7.3 and 7.5 of Network Fetch call HTTPS upgrade fallback in case of a network error. If I'm reading it correctly, streaming happens after step 12, so these steps aren't streaming related. Though, Step 7.5 seems to apply to POST requests which don't get upgraded, so it's not 100% clear to me that it should fallback. 

TLS client cert dialog indicates that the server is at least serving TLS with a valid config, so I don't think it makes sense to fallback there.

The abort/cancelled case does not fallback.

Reply:
It looks like you haven't undone the change here.